bin/run-docker

```diff
@@ -26,9 +26,13 @@ tar  -cf "${XFER_DIR}/kaleidoscope.tar"  \
 (cd "${BOARD_HARDWARE_PATH}/keyboardio" && tar  -cf "${XFER_DIR}/bundle.tar" \
 	--exclude .git --exclude avr/libraries/Kaleidoscope .)
 
+if [ -z "${DOCKER_RUN_NON_INTERACTIVE}" ]; then
+    DOCKER_RUN_INTERACTIVE_OPTS="-it"
+fi
+
```

Comment:
Does this work for you?

```
if [[ -t 0 ]]; then
    : "${DOCKER_RUN_INTERACTIVE_OPTS:=--tty --interactive}"
fi

```

Reply:
I could have sworn I tried that, but apparently not; your version works perfectly for me.